packages/react-chat/src/utils/chat.ts

```diff
@@ -1,5 +1,5 @@
 export const createPlaceholderMethods = (createMessage: (method: string) => string): Omit<VoiceflowChat, 'load' | 'destroy'> => {
-  const noopWarn = (method: string) => () => console.warn(createMessage(method));
+  const noopWarn = (method: string) => (): any => console.warn(createMessage(method));
```

Comment:
casting result as `any` so it can be a placeholder for functions with a signature like `() => Promise<void>`